src/exec.ts

```diff
@@ -18,7 +19,8 @@ export class Exec {
 
     // TODO: make command an array and support multiple args
     public async exec(namespace: string, podName: string, containerName: string, command: string,
-                      stdout: stream.Writable | any, stderr: stream.Writable | any, stdin: stream.Readable | any,
+                      stdout: stream.Writable | null, stderr: stream.Writable | null, stdin: stream.Readable | null,
+                      statusCallback: (status: V1Status) => void,
```

Comment:
Is this a breaking change for clients?  Is there anything we can do to make it non-breaking, e.g. make it the last parameter and optional?  (We seem to handle it being `null`, so could also handle it being `undefined`.)

Reply:
Updated so that it's not a breaking change.